core/src/main/res/values/strings.xml

```diff
@@ -243,6 +243,7 @@
     <string name="download_error_file_type_type">File Type Error</string>
     <string name="download_error_forbidden">Forbidden</string>
     <string name="download_canceled_msg">Download canceled</string>
+    <string name="download_wrong_size">Download does match expected size<s/tring>
```

Comment:
```suggestion
    <string name="download_wrong_size">The server connection was lost before completing the download</string>
```

I think a message like this would be easier to understand for users, even if it is not 100% accurate.